workspaces/lightspeed/plugins/lightspeed-backend/src/plugin.ts

```diff
@@ -53,6 +53,10 @@ export const lightspeedPlugin = createBackendPlugin({
           path: '/health',
           allow: 'unauthenticated',
         });
+        http.addAuthPolicy({
+          path: '/conversations',
+          allow: 'unauthenticated',
+        });
```

Comment:
can we remove this to make this a protected route?

Reply:
Thanks for catching it! This was added for testing purpose, and forgot to be removed.